packages/vendor-core/package.json

```diff
@@ -52,9 +52,9 @@
     "history": "^4.7.2",
     "humanize-duration": "3.27.0",
     "ipaddr.js": "~1.2.0",
-    "jquery": "~2.2.4",
+    "jquery": "^3.7.1",
     "jquery-ujs": "~1.2.0",
-    "jquery.cookie": "~1.4.1",
+    "js-cookie": "3.0.0",
```

Comment:
Do we need this here if foreman core depends on it? I'm not sure where we are with the whole removal of foreman-js.

Reply:
nope, forgot to remove it. removed now. were still using foreman-js